app/controllers/staff/organisations_controller.rb

```diff
@@ -12,14 +12,19 @@ def show
 
     @organisation_presenter = OrganisationPresenter.new(organisation)
 
-    fund_activities = policy_scope(Activity.funds).includes(:organisation).where(organisation: organisation).order("created_at ASC")
-    @fund_activities = fund_activities.map { |activity| ActivityPresenter.new(activity) }
+    if organisation.service_owner
+      fund_activities = policy_scope(Activity.funds).includes(:organisation).order("created_at ASC")
+      programme_activities = policy_scope(Activity.programme, policy_scope_class: ProgrammePolicy::Scope).includes(:organisation).order("created_at ASC")
```

Comment:
Hi @lozette! I was having a look at your PR (I'm practising code review today :-) ) and I have a question about the line above. Why do funds not have their own class `FundPolicy::Scope` ? Is it because they are the main "wrappers" for programmes and projects?

Reply:
Uhhh do you know what I have absolutely no idea! The change was from @mec 's commit here https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/commit/c3684969dc312580bd7be6ce47594f7f9e30da28#diff-432213d9aea29cede7ca815df9804248 - @mec can you shed some light?

Reply:
Well, funds do have a policy, and probably should use their own policy scope here - otherwise they are using the base `ActivityPolicy` which may not be the best. 

Looking at `ActivityPolicy` we have a lot of ugly logic in there that using a class would remove, it also looks like the scope includes all funds for administrator users (everyone) which may not be correct whereas the `FundPolicy` includes funds only for BEIS users, which seems more resaonable.

I haven't looked at this PR apart from your question so I don't know if this line introduces this issue or if we are just following what has come berfore, but either way it's a great spot and a great question! :D

Reply:
I was following what came before but I'll amend the PR to use the Fund `policy_scope_class` and see if anything breaks!!